go.mod

```diff
@@ -16,6 +16,7 @@ replace (
 
 	github.com/rancher/rancher/pkg/apis => ./pkg/apis
 	github.com/rancher/rancher/pkg/client => ./pkg/client
+	github.com/rancher/shepherd => github.com/josh-diamond/shepherd v0.0.0-20240214234402-e569c4ded1bd
```

Comment:
Reminder that this is pointing to your forked repo. Once testing is done, please point to the official repo and not your forked repo.